.gitignore

```diff
@@ -3,6 +3,7 @@
 tmp/_output
 tmp/_test
 _output
+env.sh
```

Comment:
is there any reason this can't be dumped into `$CLUSTER_DIR`?

Reply:
good point, done.